plugins/azure/frontdoor/frontDoorWafDefaultRuleSet.js

```diff
@@ -0,0 +1,69 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Front Door WAF Latest Default Rule Set',
+    category: 'Front Door',
+    domain: 'Content Delivery',
+    severity: 'Low',
+    description: 'Ensures that Azure Front Door WAF is using the latest Azure managed default rule set with action set to block.',
+    more_info: 'Azure-managed rule sets provide an easy way to deploy protection against a common set of security threats. Azure updates these rules as needed to protect against new attack signatures. ',
+    recommended_action: 'Modify the Front Door WAF policy and add latest default rule set.',
```

Comment:
```suggestion
    description: 'Ensures that Azure Front Door WAF is using the latest Azure managed default rule set with action set to block.',
    more_info: 'Azure-managed rule sets provide an easy way to deploy protection against a common set of security threats. Azure updates these rules as needed to protect against new attack signatures. ',
    recommended_action: 'Modify the Front Door WAF policy and add latest default rule set.',
```
should mention that this is only for premium policies